server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/model/message/view/MessageExtractorHelper.java

```diff
@@ -0,0 +1,74 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.jmap.draft.model.message.view;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Optional;
+
+import javax.inject.Inject;
+
+import org.apache.james.jmap.draft.utils.HtmlTextExtractor;
+import org.apache.james.mailbox.exception.MailboxException;
+import org.apache.james.mime4j.dom.Message;
+import org.apache.james.mime4j.stream.MimeConfig;
+import org.apache.james.util.mime.MessageContentExtractor;
+import org.apache.james.util.mime.MessageContentExtractor.MessageContent;
+
+import com.google.common.base.Strings;
+
+public class MessageExtractorHelper {
+    private final MessageContentExtractor messageContentExtractor;
+    private final HtmlTextExtractor htmlTextExtractor;
+
+    @Inject
+    public MessageExtractorHelper(MessageContentExtractor messageContentExtractor, HtmlTextExtractor htmlTextExtractor) {
+        this.messageContentExtractor = messageContentExtractor;
+        this.htmlTextExtractor = htmlTextExtractor;
+    }
+
+    public Message parse(InputStream messageContent) throws MailboxException {
```

Comment:
I don't like that method because it hides a builder.

Mime4J parsing is not that complex that it should be abstracted away. 

Duplication of that method is acceptable to me.